### PR TITLE
Remove obsolete or otherwise unnecessary code

### PR DESCRIPTION
**Pull request by cmutel**
Some stuff should be in Brightway, not here

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,9 +30,7 @@
     install_requires=[
         "appdirs",
         "bw_processing",
-        "fasteners",
         "peewee>=3.9.4",
-        "wrapt",
```
Comment (baali) [inline setup.py:35]: We can't remove `wrapt` we are using it for [tests](https://github.com/baali/bw_projects/blob/main/tests/projects.py) and creating `tempdir` during the tests.

**Comment by cmutel** [comment]
OK, this can be changed. But eventually we can remove this, I think we can have a normal decorator for tests, `bw2data` will continue to have the `bw2tests` decorator but we shouldn't use it here as we don't want a dependency on `bw2data`.

File: setup.py (hunk 1 above)
Comment (baali) [reply setup.py:35]: I am [shifting](https://github.com/baali/bw_projects/pull/13/commits/358d1ceaa1904e1e1fba88fca45f5df31a61a4db) to other approach of handling `tempdir` and cleaning them up.

**Comment by baali** [comment]
Thank you Chris for the PR :pray:

**Review by baali — APPROVED** [review]

**Pull request merged**